### ISSUE TITLE
High Res Training

### ISSUE THREAD

**Issue by siddu9501** [Rudrabha/Wav2Lip#130]
I was able to train a high res syncnet well; 
 
After adding conv blocks in Wav2Lip encoder / decoder, 
When training the wav2lip model with this architecture, The model works well generating somewhat blurry images with fair lipsync. 
L1: 0.04 ; Sync Loss: 0.4;

However, introducing the syncnet at this point pushes the generated frame to look like the reference frame instead of the ground truth frame along with some artifacts. 
L1: 0.1 Sync Loss: 0.89;

Does the architecture scale well into higher resolution?

**Comment by kartikeyanegi** [comment]
I am trying to train the model on a custom high resolution dataset. Since you have already tried it out, it would be helpful if you can answer some questions for me. 1)The models trained on low resolution 96px detected faces. For higher resolution, what size did you increase the resolution too? 2) How many hours of data did you use? 3) What layers (conv blocks) did you add to get the 1-d embedding to the encoder? 4)How long before the loss converged to ~.20 for syncnet (as in how many epochs or steps) 5) Any other changes you made in the learning rate, optimizer etc?

**Comment by prajwalkr** [comment]
> L1: 0.1 Sync Loss: 0.89;

Use a lower learning rate when introducing the sync loss (divide by 10 at least).

**Comment by siddu9501** [comment]
I used a lower learning rate, the sync loss is still not consistent during training and goes from 1.x to 0.4 for different samples. I don't see improvement in the sync loss after adding the syncnet. The L1 loss however is consistent. 
The pre-trained model you have uploaded was used as the baseline for my high-res syncnet model and both models give similar losses for my video dataset. 

I'm also trying to understand the 0-indexing -- 1-indexing problem a bit more in detail to see if it could be causing a problem here.

**Comment by oraculseed** [comment]
> I was able to train a high res syncnet well;
> 
> After adding conv blocks in Wav2Lip encoder / decoder,
> When training the wav2lip model with this architecture, The model works well generating somewhat blurry images with fair lipsync.
> L1: 0.04 ; Sync Loss: 0.4;
> 
> However, introducing the syncnet at this point pushes the generated frame to look like the reference frame instead of the ground truth frame along with some artifacts.
> L1: 0.1 Sync Loss: 0.89;
> 
> Does the architecture scale well into higher resolution?

Perhaps the problem is in the conv layers. What code did you change ?
